src/components/Source.js

```diff
@@ -1,13 +1,27 @@
 import React from 'react';
-
-function Source() {
-  return(
-  <div className="source" style={{backgroundColor:'#0093D0', padding:'-10%'}}>
-  <h2>Sources</h2>
-  <h4>Source Title</h4>
-  <p> <a href="https://classroomswithoutborders.org/" target="_blank" rel="noopener noreferrer">www.source1.com</a> </p>
-  </div>
-  )
+import { Component } from 'react';
+import constants from '../styling/Constants';
+ 
+class  Source extends Component {
```

Comment:
change to functional component if possible, since that's the syntax we're using for other components as well, like:
```
const Source = (props) => {
    ...
    return (...)
}
```